validator/client/beacon-api/beacon_block_proto_helpers.go

```diff
@@ -0,0 +1,352 @@
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to get attestation 2")
+		}
+
+		attesterSlashings[index] = &ethpb.AttesterSlashing{
+			Attestation_1: attestation1,
+			Attestation_2: attestation2,
+		}
+	}
+
+	return attesterSlashings, nil
+}
+
+func convertAttestationToProto(jsonAttestation *apimiddleware.IndexedAttestationJson) (*ethpb.IndexedAttestation, error) {
+	if jsonAttestation == nil {
+		return nil, errors.New("indexed attestation is nil")
+	}
+
+	attestingIndices := make([]uint64, len(jsonAttestation.AttestingIndices))
+
+	for index, jsonAttestingIndex := range jsonAttestation.AttestingIndices {
+		attestingIndex, err := strconv.ParseUint(jsonAttestingIndex, 10, 64)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to parse attesting index `%s`", jsonAttestingIndex)
+		}
+
+		attestingIndices[index] = attestingIndex
+	}
+
+	signature, err := hexutil.Decode(jsonAttestation.Signature)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to decode attestation signature `%s`", jsonAttestation.Signature)
+	}
+
+	attestationData, err := convertAttestationDataToProto(jsonAttestation.Data)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get attestation data")
+	}
+
+	return &ethpb.IndexedAttestation{
+		AttestingIndices: attestingIndices,
+		Data:             attestationData,
+		Signature:        signature,
+	}, nil
+}
+
+func convertCheckpointToProto(jsonCheckpoint *apimiddleware.CheckpointJson) (*ethpb.Checkpoint, error) {
+	if jsonCheckpoint == nil {
+		return nil, errors.New("checkpoint is nil")
+	}
+
+	epoch, err := strconv.ParseUint(jsonCheckpoint.Epoch, 10, 64)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to parse checkpoint epoch `%s`", jsonCheckpoint.Epoch)
+	}
+
+	root, err := hexutil.Decode(jsonCheckpoint.Root)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to decode checkpoint root `%s`", jsonCheckpoint.Root)
+	}
+
+	return &ethpb.Checkpoint{
+		Epoch: types.Epoch(epoch),
+		Root:  root,
+	}, nil
+}
+
+func convertAttestationsToProto(jsonAttestations []*apimiddleware.AttestationJson) ([]*ethpb.Attestation, error) {
+	attestations := make([]*ethpb.Attestation, len(jsonAttestations))
+
+	for index, jsonAttestation := range jsonAttestations {
+		if jsonAttestation == nil {
+			return nil, errors.Errorf("attestation at index `%d` is nil", index)
+		}
+
+		aggregationBits, err := hexutil.Decode(jsonAttestation.AggregationBits)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to decode aggregation bits `%s`", jsonAttestation.AggregationBits)
+		}
+
+		attestationData, err := convertAttestationDataToProto(jsonAttestation.Data)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to get attestation data")
+		}
+
+		signature, err := hexutil.Decode(jsonAttestation.Signature)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to get attestation signature `%s`", jsonAttestation.Signature)
```

Comment:
```suggestion
			return nil, errors.Wrapf(err, "failed to decode attestation signature `%s`", jsonAttestation.Signature)
```
Just for uniformity.